server/pschema.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE films (
+    code        char(5) CONSTRAINT firstkey PRIMARY KEY,
+    title       varchar(40) NOT NULL,
+    did         integer NOT NULL,
+    date_prod   date,
+    kind        varchar(10),
```

Comment:
My only comment is if you capitalize all SQL syntax, be consistent. CHAR(5) CONSTRAINT firstkey PRIMARYKEY, VARCHAR(40) and etc.

Reply:
lol, this is the example script I found googling, I'll remove it.